src/Tribe/Template/Month.php

```diff
@@ -156,22 +156,13 @@ public function __construct( $args = null ) {
 			$this->first_grid_date = self::calculate_first_cell_date( $this->requested_date );
 			$this->final_grid_date = self::calculate_final_cell_date( $this->requested_date );
 
-			$args = array_merge( $args, array(
-				'fields'         => 'ids',
-				'start_date'     => $this->first_grid_date,
-				'end_date'       => $this->final_grid_date,
-				'post_status'    => array( 'publish' ),
-				'posts_per_page' => - 1,
-			) );
 
 			if ( is_user_logged_in() ) {
 				$args['post_status'][] = 'private';
 			}
```

Comment:
Looks like you missed incorporating this into the new `set_events_in_month()` query.